sdk/utils.ts

```diff
@@ -1,12 +1,38 @@
 import fetch, { RequestInit, Response } from 'node-fetch';
+import { ITraitConfig } from './types';
 // @ts-ignore
 if (typeof fetch.default !== 'undefined') fetch = fetch.default;
 
-export function generateIdentitiesData(identifier: string, traits: { [key: string]: any }) {
-    const traitsGenerated = Object.entries(traits).map(trait => ({
-        trait_key: trait[0],
-        trait_value: trait[1]
-    }));
+type Trait = {
+    [key: string]: ITraitConfig | any
+}
+
+export function isTraitConfig(trait: Trait): trait is Trait {
+    return !!trait && typeof trait == 'object' && trait.value !== undefined;
+}
```

Comment:
There's a number of problems with this type guard.

1. Judging from its name, its signature should be different:
```suggestion
export function isTraitConfig(traitValue: any): traitValue is TraitConfig {
    return !!traitValue && typeof traitValue == 'object' && traitValue.value !== undefined;
}
```

2. Judging from its usage it should accept a key-value pair, but its logic only checks the value.

Reply:
1) Renamed
2) Types corrected